<!doctype html>
<!-- 
##############################################################
Ignore this boilerplate if you're just trying to edit the text.
Skip to the part that says 'The real text begins here'
##############################################################

Based on this theme: https://github.com/broccolini/dinky , which mentioned that attribution is appreciated. Thanks, broccolini!
-->
<html lang="en">
  <head>
    <base target="_blank">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Relaxation Sensors Appendix</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/prism.css">
    <!--[if lt IE 9]>
    <script src="javascript/html5shiv/html5shiv.js"></script>
    <![endif]-->
    <!--[if lt IE 8]>
      <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
    <![endif]-->
    <script src="javascript/scale-fix/scale.fix.js"></script>
    <script src="javascript/python-highlighting/prism.js"></script>
    <script async  src="javascript/Minimal-MathJax/MathJax.js?config=TeX-AMS_CHTML"></script>
    <script src="javascript/update_figures.js"></script>
    <script src="javascript/reference_list/reference_list.js"></script>
    <script src="javascript/google-analytics/analytics.js"></script>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1 class="header">Relaxation Sensors</h1>
        <ul>
          <li class="download"><a class="buttons" href="https://github.com/AndrewGYork/relaxation_sensors/archive/refs/heads/main.zip">Download ZIP</a></li>
          <li><a class="buttons github" href="https://github.com/AndrewGYork/relaxation_sensors">View On GitHub</a></li>
          <li><a class="buttons pdf" href="./Relaxation_Sensors_Appendix.pdf">Download PDF</a></li>
        </ul>
        <p class="header">This project is maintained by <a class="header name" href="https://github.com/marimar128">Maria Ingaramo</a> in the <a class="header name" href="http://andrewgyork.github.io/">York lab</a>, and was funded by <a class="header name" href="https://www.calicolabs.com/">Calico Life Sciences LLC</a></p>
      </header>
<!-- 
##############################################################
The real text begins here.
##############################################################
 -->
<section>
<h1>Appendix</h1>

<h3 class="onlyprint">
Note that this is a limited PDF or print version; animated and interactive figures are disabled. For the full version of this article, please visit <a href="https://andrewgyork.github.io/relaxation_sensors">https://andrewgyork.github.io/relaxation_sensors</a></h3>

<h2>Relaxation Sensors</h2>
<p><a href="./index.html" title="Main text" target="_self">Back to the main text</a></p>


<h3 id="mitochondria_in_mammalian_cells">
Mammalian cell imaging</h3>
<p>
We prepared COS7 cells transiently expessing pH-Countdown targeted to the mitochondrial matrix using a COX8A presequence. We imaged the samples in live cell imaging solution (ThermoFisher, A14291DJ) + 5 mM glucose at 35&deg;C on a DMI8 widefield microscope controlled with micromanager using a 63x 1.4 NA oil objective, a 515/30 nm emission filter, and a SpectraX LED illumination source with 395/25 or 470/24 nm excitation filters. Samples were prepared in 8 well #1.5 glass chambers (Cellvis) and the temperature was maintained using an Okolabs enclosure system. Data was acquired in the '<a href="#activation_due_to_measurement" target="_self">disruptive</a>' regime with a 2 second decay period after activation. We induced uncoupling (which is expected to lower mitochondrial pH) by adding a final concentration of 3 &mu;M CCCP. The relationship between pH values and relaxation rates at this temperature were calibrated using cells permeabilized with 10 &mu;M nigericin in 200 &mu;L Spexyte buffers (AAT Bioquest) + 300 &mu;L Carmody buffers supplemented with 100 mM KCl at pH 6.5, 7, 7.5 and 8.
</p>
<p>These cultured cells are transparent, with almost no detectable scattering or autofluoresence, so they're well suited for dual-color intensity sensors like <a href="http://doi.org/10.4236/abb.2011.23021">ratiometric pHluorin</a> or <a href="https://doi.org/10.1152/ajpheart.1998.275.6.H1937">SNARF</a>. Unfortunately for relaxation sensors, the mitochondria move quickly, sometimes changing position by several pixels during our 2 s relaxation interval. Therefore we used whole-cell relaxation ratios as a simple estimator of cell-averaged mitochondrial pH, which is robust against motion artifacts.
</p>
<figure id="mitochondria_phcountdown_figure">
<img src="./images/Appendix_cos7_mitochondria_ph_figure/1_data_animation.gif" alt="An timelapse of mitochondria tagged with pH-Countdown showing strong pH change upon addition of CCCP">
 <figcaption><strong>Figure: Mitochondrial pH decrease upon addition of the uncoupler CCCP. <em>
Left panel:</em></strong> Timelapse images of mitochondria tagged with pH-Countdown. Pseudocolor indicates cell-averaged pH (calculated via relaxation ratio), which decreases upon addition of CCCP. <strong><em>Right panel: </em></strong> Calculated whole-cell relaxation ratio vs. time; colored horizontal stripes correspond to colored boxes in left panel, showing the relationship between relaxation ratio and pH. Sensor photobleaching per photoactivation-relaxation-photodeactivation measurement cycle was ~1%.
 </figcaption>
</figure>

<h3 id="ph_countdown_in_yeast">
Yeast</h3>
<p>
Yeast (<em>S. cerevisiae </em>BY4741) expressing pH-Countdown from a p416 plasmid and <a href="https://doi.org/10.7939/R33R0Q22X">pHlorina</a> (a pH-sensitive, ratiometric red fluorescent protein derived from mApple) from a p413 plasmid were grown in synthetic media without URA  or HIS for 24 hours. 6150 µL of fresh media were inoculated with 50 µL of the overnight growth, and sampled every 75 minutes for OD<sub>600</sub> and fluorescence measurements. To measure pH-Countdown relaxation ratios and pHlorina fluorescence excitation ratios, we used a Leica DMI8 wide-field microscope with a SOLA light engine, a 40X 0.85 NA objective, a 1.6X mag changer, and DAPI, GFP, TXR cubes, and a custom cube (Semrock FF01-440/40, FF470-Di0, FF01-585/40). We measured relaxation ratios in the '<a href="#activation_due_to_measurement" target="_self">disruptive</a>' regime, with a spontaneous decay interval of 15 seconds. Calibration was done in permeabilized yeast in Carmody buffer, pH 6 to 8, with 0.1% digitonin. 
</p>
<p> 
Cytosolic pH in yeast is known to be correlated with growth rate. pH-Countdown reveals a decrease from pH ~7.4 to ~6.8 as the culture reaches saturation, which matches the trend reported by pHlorina. 
</p>

<figure id="yeast_ph_figure">
<img src="./images/yeast/yeast1.png" alt="Yeast ph figurefigure" id="yeast_ph">
<table class="figure_controls">
  <tr>
  <td>Plot: </td><td>
    <select id="yeast_ph_" onchange="update_yeast_ph_figure()">
    <option value="yeast1.png" selected="selected">pH-Countdown</option>  
    <option value="yeast2.png">pH-Countdown and pHlorina</option> 
  </select>
</td>
    </tr>
</table>
 <figcaption><strong>Figure: pH-Countdown and pHlorina agree quantitatively in live yeast.</strong><em><strong> Left panel:</strong></em> calibration curve for pH-Countdown. <strong><em>Right panel:</em></strong> pH-Countdown or pH-Countdown plus pHlorina traces during yeast growth.
 </figcaption>
</figure>

<h3 id="temperature_dependence">
Relaxation rate temperature dependence</h3>
<p>
We imaged Ni-NTA (cOmplete resin, Roche) agarose beads with bound pH-Countdown in the '<a href="#activation_due_to_measurement" target="_self">non-disruptive</a>' mode on a DMI8 wide-field microscope controlled with Micro-Manager using a 10x 0.23 NA air objective, a 515/30 nm emission filter, and a SpectraX LED illumination source with 395/25 or 470/24 nm excitation filters. Samples were in Carmody buffer and temperature was maintained using a VAHEAT control system. 
</p>

<figure id="temperature_sensitivity_figure">
<img src="./images/temperature_sensitivity_figure/25.png" alt="Temperature sensitivity figure" id="temperature_sensitivity">
<table class="figure_controls">
  <tr>
  <td>Plot: </td><td>
    <select id="temperature_sensitivity_" onchange="update_temperature_sensitivity_figure()">
    <option value="25.png" selected="selected">25&deg;C</option>  
    <option value="30.png">30&deg;C</option>
    <option value="35.png">35&deg;C</option>  
    <option value="40.png">40&deg;C</option> 
    <option value="ph6.png">pH 6</option>
    <option value="ph6p5.png">pH 6.5</option>   
    <option value="ph7.png">pH 7</option> 
    <option value="ph7p5.png">pH 7.5</option>
    <option value="ph8.png">pH 8</option>  
  
  </select>
</td>
    </tr>
</table>
 <figcaption><strong>Figure: pH-countdown temperature dependence</strong>
 </figcaption>
</figure>

<h3 id="off_target_sensing">
Relaxation rate dependence on buffer components</h3>
<p>
Ni-NTA (cOmplete resin, Roche) agarose beads with bound pH-Countdown were imaged on a wide-field microscope in different solutions using the '<a href="#activation_due_to_measurement" target="_self">non-disruptive</a>' acquisition mode. We measured NaCl, KCl, MgCl<sub>2</sub>, MgSO<sub>4</sub>, glycerol, DTT, and peroxide dependencies in 45 mM sodium phosphate buffer, pHed to 8 at room temperature, by mixing solutions from concentrated stocks. CaCl<sub>2</sub> was titrated in HEPES buffer, pH 8, to avoid calcium phosphate precipitation. The above mentioned experiments were carried out at 30&deg;C using an Okolab temperature controlled enclosure. Screening of buffering agent effects was done at 25&deg;C with a VAHEAT temperature control system, using buffers with pH adjusted at 25&deg;C.
</p>

<figure id="buffer_sensitivity_figure">
<img src="./images/buffer_sensitivity_figure/1_phosphate_nacl.png" alt="Buffer sensitivity figure" id="buffer_sensitivity">
<table class="figure_controls">
  <tr>
  <td>Confounding factor: </td><td>
    <select id="buffer_sensitivity_" onchange="update_buffer_sensitivity_figure()">
    <option value="1_phosphate_nacl.png" selected="selected">NaCl</option>  
    <option value="2_phosphate_kcl.png">KCl</option>
    <option value="3_phosphate_mgcl2.png">MgCl_2</option>  
    <option value="4_phosphate_mgso4.png">MgSO_2</option> 
    <option value="5_phosphate_dtt.png">DTT</option>
    <option value="6_phosphate_peroxide.png">Peroxide</option>   
    <option value="7_phosphate_sorbitol.png">Sorbitol</option> 
    <option value="8_glycerol.png">Glycerol</option>
    <option value="9_cacl2.png">CaCl_2</option>  
    <option value="0_buffers.png" >Buffer</option>    
  
  </select>
</td>
    </tr>
</table>
 <figcaption><strong>Figure: pH-countdown sensitivity to common buffer components</strong>

 </figcaption>
</figure>

<h3 id="robust_against_perturbations">
Robustness of pH-Countdown relaxation rates against scattering, background and attenuation</h3>
<p>
To investigate robustness against scattering, attenuation and background, we imaged 5 &mu;L drops of 7 mg/mL pH-Countdown in Carmody buffer sandwiched between a slide and a coverslip, contained using 0.5 mm thick silicone spacers with 6.6 mm diameter holes. We covered the sample with ground glass (DG100X100-120, DG100X100-220, DG100X100-600 or DG100X100-1500, Thorlabs) to assess the effect of scattering, with neutral density filters (NE201B, NE202B, NE203B or NE204B, Thorlabs) to assess the effects of attenuation, or with hybridization chambers (70333-50 and 71862-01, EMS) filled with different concentrations of fluorescein in PBS to assess the effects of fluorescent background. Temperature was set to 25&deg;C using an Okolab stage-top chamber, and imaged in the '<a href="#activation_due_to_measurement" target="_self">non-disruptive</a>' mode on our custom <a href="#chicken_blaster" target="_self">fluorescence photography</a> setup. As we hoped, measured relaxation rates are robust against scattering, background, and attenuation, which makes relaxation sensors particularly well suited for imaging in tissues.
</p>
<figure id="robustness_quantitation_figure">
<img src="./images/robustness_quantitation_figure/3_fluorescein.png" alt="Robustness quantitation figure" id="robustness_animation">
<table class="figure_controls">
  <tr>
  <td>Confounding factor: </td><td><select id="robustness_animation_" onchange="update_robustness_quantitation_figure()">
    <option value="1_scatter.png">Scattering</option> 
    <option value="2_attenuation.png">Attenuation</option>
    <option value="3_fluorescein.png" selected="selected">Background</option>   
  </select>
</td>
    </tr>
</table>
 <figcaption><strong>Figure: Robustness of pH-countdown against scattering, attenuation and background. Each family of curves is a distinct pH (6, 6.5, 7, 7.5, 8), as in Figure 2 of the main text.</strong>

 </figcaption>
</figure>

<h3 id="mutation_history">
Notes on the directed evolution of Countdown</h3>

<p>We originally started this journey through mutational space because we wanted a <a href="https://doi.org/10.1038/ncomms9184">"two-step" fluorophore</a> that rapidly spontaneously deactivates. We started with Padron because that's what we'd been using. In hindsight, Padron had some important flaws. For example, its oligomerization state strongly affects its contrast ratio, and its maturation is extremely slow. Padron also takes more than two hours to spontaneosly deactivate. We were in for a long journey. Over the years, we learned many important things about mutagenesis, screening, and the protein itself, including:
</p>

<h4>
How to make libraries:
</h4>
<ul>
 <li>
 Random mutagenesis (e.g. <a href="https://www.agilent.com/en/product/mutagenesis-cloning/mutagenesis-kits/random-mutagenesis-kits/genemorph-ii-233115">Agilent Genemorph II</a>) was suprisingly useless in our hands, possibly because the resulting library sizes were much bigger than our screening throughput. Perhaps if we'd had a flow-cytometry-based screen with throughputs of 10<sup>6</sup> mutants/day, this would have worked better, but our plate-based screening throughput was closer to 10<sup>4</sup> mutants/day. As a result, depending on the degree of mutagenesis, we ended up screening libraries that either had mostly unmutated protein, or mostly nonfluorescent over-mutated variants.
 </li>
 <li>
 Semi-random/semi-rational mutagenesis worked great, at first. We would "rationally" pick a location on our protein, make a small library of all 20 mutants at that location, and screen a plate of those ~20 mutants. We'd repeat this process for several locations, before picking a new champion for further mutagenesis. We produced our first ~5 mutations this way.
</li> 
 <li>
Semi-random mutagenesis at every residue worked even better. We ordered 448 primers (~$2000), and used them to make all single mutants at all locations (4480 mutants). We'd then pool the mutants into a single library, spread the library across ~10 plates and screen for a winner. After selecting a winner, it costs ~$60 to update the primers for the new champion.
  <ul>
    <li>We were initially reluctant to spend $2000 on primers, but this works much better than anything else we tried, and is very cheap compared to the cost of labor and the value of our time.</li>
    <li>The size of this library (10<sup>4</sup> mutants) matches the rough daily throughput of our plate-based screen. In general, the size of one's libraries should relate to the throughput of one's screen.</li>
    <li>We had great results when we simply reused the same primers for multiple consecutive rounds, rather than updating primers at each round. This probably selected against multiple consecutive nearby mutations, but maybe this is good?</li>
    <li>We also had great results when we would pick multiple winners, rather than a single winner at each round; we often picked and pooled ~10 winners as the starting point for the next round of mutagenesis. If you're more concerned with progress than insight, you don't even have to sequence your winners, you can just keep mutating and screening for several iterations.</li>
  </ul>
  </li>
</ul>

<h4>
How to screen libraries:
</h4>

<ul>
  <li>
Screening <em>E. coli</em> colonies on plates worked well and wasn't too difficult. However, the fluorophore is probably at high concentration inside the <em>E. coli</em>, and we worried this might select for oligomerization-dependent photophysics rather than a high-performance monomer.
  </li>
  <li>
We eventually switched to lysing <em>E. coli</em> onto nitrocellulose, which lets us <a href=#countdown_mutagenesis target="_self">control the chemical environment</a> of the fluorophore. We also suspect this lowers the effective concentration of the fluorophore, and perhaps selects against oligomerization-dependent photophysics.
  </li>
  <li>
Our first screen had a much smaller field of view (&lt;1 cm<sup>2</sup>), and used an XY stage to tile a full plate. 1-2 years later, we switched to a much bigger XY stage that let us tile multiple (~10) plates. Since the fluorophore's spontaneous switching was still quite slow at the time, most of the screening time was spent waiting, and this increased our throughput by an order of magnitude.
  </li>
  <li>
This past year, we switched to <a href="#chicken_blaster" target="_self">our current screen</a> with a full-plate (~10 cm) field-of-view, which is much nicer to use. Since the protein decays in seconds now instead of hours, it's human-tolerable to manually switch one plate at a time for ~10 plates in a row (~1 hour). Since we plan to further accelerate the off-switching of pH-Countdown, we should probably add a large XY stage to this screen so we can load ~10 plates at a time. In hindsight, we should have upgraded our screen much sooner.
  </li>
  <li>
It's not obvious that pooling the full library into one pool is optimal. Arguably, if we're going to screen ~10 plates, pooling into ~10 sub-libraries, rather than a single large library, would potentially yield more immediate insight into which mutation locations have which photophysical effects.
  </li>
</ul>

<h4>
Some things we learned about the protein:
</h4>

<ul>
  <li>
Padron's maturation is extremely slow, which turned out to be a major unexpected bottleneck. Our first mutation (at position 173) was a huge improvement, which made maturation go from several days to &lt;1 day.
  </li>
  <li>
Mutations that made spontaneous switching faster almost always made light-driven switching faster too. Maybe once we found a mutation that seemed to affect photoswitching rates differently from how it affected spontaneous switching, but we did not stop to verify the behavior.
  </li>
  <li>
At one point, we failed to find a mutant we considered "better" by screening all the single mutants. In desperation, we made a double-mutant library (with &gt;10<sup>6</sup> members), and screened unhappily until we found an improved double mutant. In hindsight, it's surprising how well a simple nearest-neighbor hill-climbing search strategy worked, and fortunate that we only had to do this once.
  </li>
  <li>
We were fairly flexible in our definition of "better" when selecting candidates for the next round of mutagenesis. We mostly selected for switching speed, but we also wanted to preserve on-state brightness, off-state darkness, cyan-driven on-switching, violet-driven off-switching, and equilibrium to the off-state. Sometimes we'd spend a round or two to improve brightness or equilibrium without improving switching speed. We would often accept a small drawback in exchange for a large advantage, and it often took several iterations to produce a strict improvement.
  </li>
  <li>
Most of our screening was done without temperature or pH control. In hindsight, it's miraculous that this worked as well as it did.
  </li>
</ul>

<p>
The figure below shows (photo)switching kinetics for 17 representative mutants from our ~30-iteration mutational series, beginning with Padron and ending with pH-Countdown, taken in a single batch. This work spanned many years and multiple transitions in our careers, but we're happy to share whatever information we have access to, if someone is interested in more detail.
</p>
<img src="./images/Appendix_mutant_history/shorter_time_box.png" alt="Mutants generated during evolution of pH-countdown" id="mutation_history_photo">

<h3 id="countdown_mutagenesis">
Screening for pH-sensitive relaxation</h3>
<p>
We created libraries for saturated mutagenesis at every residue using rolling circle PCRs with NNK primers for every coding residue on a pQE31-Countdown plasmid. PCRs were pooled, DPN-I digested, and transformed into BL21(DE3) <em>E. coli</em>. After overnight growth and one day at room temperature, we blotted the plates onto nitrocellulose filters. The filters were dipped in liquid nitrogen three times, submerged in Carmody buffer at pH 7.5 with 10 &mu;M CCCP, imaged on our custom <a href="#chicken_blaster" target="_self">fluorescence photography</a> setup, and then imaged again after transfer  to Carmody buffer at pH 5.5 with 10 &mu;M CCCP. Colonies that displayed the largest rate difference during the first 20 seconds of decay were selected from the original agar plates after a second overnight growth. The winners were subjected to subsequent saturated mutagenesis rounds.
</p>


<h3 id="calcium_sensing_library">
Library construction for the calcium-sensitive relaxation sensor</h3>
<p>
We constructed libraries screening for linker length and residue type using Gibson assembly. The pH-Countdown insert and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3560286/">GECO 1.2</a> plasmid were PCR amplified using the primers indicated below. The screening procedure was similar to the <a href="#countdown_mutagenesis" target="_self">pH screening method</a>, except that the nitrocellulose filters were washed in 50 mM Tris pH 8 + 1 mM EGTA, and imaged in the same buffer with either 100 &mu;M EGTA or 3 mM CaCl<sub>2</sub>.
</p>
<table>
  <tr>
    <th>Primer Name</th>
    <th>Sequence</th>
  </tr>
  <tr>
    <td>2GCO_plasR</td>
    <td>TGAGCTCAGTCTTCCAATGGCACGAACAGCATGC</td>
  </tr>
  <tr>
    <td>2GCO_plasf</td>
    <td>GACCAGTTAACCGAGGAACAAATAGCTGAGTTCAAGGAAG</td>
  </tr>
  <tr>
    <td>2GCO_Of</td>
    <td>CCATTGGAAGACTGAGCTCASSNatggtgagcaagggcgaggagaac</td>
  </tr>
  <tr>
    <td>2GCO_m2f</td>
    <td>CCATTGGAAGACTGAGCTCASSNaacaacatggccgtgattaaaccagacatg</td>
  </tr>
  <tr>
    <td>2GCO_2f</td>
    <td>CCATTGGAAGACTGAGCTCASSNgccgtgattaaaccagacatgaagatcaagc</td>
  </tr>
  <tr>
    <td>2GCO_3f</td>
    <td>CCATTGGAAGACTGAGCTCASSNgtgattaaaccagacatgaagatcaagctgcg</td>
  </tr>
  <tr>
    <td>2GCO_4f</td>
    <td>CCATTGGAAGACTGAGCTCASSNattaaaccagacatgaagatcaagctgcgtatgg</td>
  </tr>
  <tr>
    <td>2GCO5_5f</td>
    <td>CCATTGGAAGACTGAGCTCASSNaaaccagacatgaagatcaagctgcgtatggaag</td>
  </tr>
  <tr>
    <td>2GCO_6f</td>
    <td>CCATTGGAAGACTGAGCTCASSNccagacatgaagatcaagctgcgtatggaaggc</td>
  </tr>
  <tr>
    <td>2GCO_8f</td>
    <td>CCATTGGAAGACTGAGCTCASSNatgaagatcaagctgcgtatggaaggctctg</td>
  </tr>
  <tr>
    <td>2GCO_10f</td>
    <td>CCATTGGAAGACTGAGCTCASSNatcaagctgcgtatggaaggctctgtaaatg</td>
  </tr>
  <tr>
    <td>2GCO_0r</td>
    <td>TGTTCCTCGGTTAACTGGTCNSScttgtacagctcgtccatggcc</td>
  </tr>
  <tr>
    <td>2GCO223r</td>
    <td>TGTTCCTCGGTTAACTGGTCNSSggcctgcctcggcagctcag</td>
  </tr>
  <tr>
    <td>2GCO220r</td>
    <td>TGTTCCTCGGTTAACTGGTCNSScggcagctcagaatgtgcttcggc</td>
  </tr>
  <tr>
    <td>2GCO219r</td>
    <td>TGTTCCTCGGTTAACTGGTCNSScagctcagaatgtgcttcggcatgctc</td>
  </tr> 
  <tr>
    <td>2GCO217r</td>
    <td>TGTTCCTCGGTTAACTGGTCNSSagaatgtgcttcggcatgctcatgcag</td>
  </tr> 
  <tr>
    <td>2GCO215r</td>
    <td>TGTTCCTCGGTTAACTGGTCNSStgcttcggcatgctcatgcagattaac</td>
  </tr> 
  <tr>
    <td>2GCO213r</td>
    <td>TGTTCCTCGGTTAACTGGTCNSSggcatgctcatgcagattaacattactgtaatc</td>
  </tr> 
  <tr>
    <td>2GCO213r</td>
    <td>TGTTCCTCGGTTAACTGGTCNSSctcatgcagattaacattactgtaatctttgtcg</td>
  </tr>           
</table>

<h3 id="elegans_imaging">
C. elegans imaging</h3>
<p>
We used a <a href="https://doi.org/10.1002/j.1460-2075.1991.tb04966.x">standard microinjection procedure</a> to introduce 25 ng/&mu;L <a href="#sequences" target="_self">Peft-3::pH-Countdown::unc-54 pUC57 plasmid</a> mixed with 20 ng/&mu;L <a href="https://www.addgene.org/vector-database/7880/">pRF4</a> into the gonads of young N2E hemaphrodite <em>C. elegans</em>. Progeny displaying the roller phenotype were inspected for sensor expression after 48 hours at 25&deg;C. For imaging, we placed the worms on 3% agarose pads containing a drop of 2 mM levamisole (to paralyze them) and covered them with a #1.5 coverslip. We imaged the samples on a DMI8 widefield microscope controlled with micromanager using a 40x 0.85 NA air objective, a 515/30 nm emission filter, and a SpectraX LED illumination source with 395/25 or 470/24 nm excitation filters.
</p>
<p>Data were acquired in the '<a href="#activation_due_to_measurement" target="_self">disruptive</a>' regime. We used a series of violet light exposures to rapidly push pH-Countdown into the inactive state, before using a series of 50 ms, 1.7 W/cm<sup>2</sup> cyan light exposures to produce an activation curve, and a single delayed cyan light exposure to measure the amount of relaxation. We calibrated the relationship between relaxation and pH by imaging Ni-NTA (cOmplete resin, Roche) agarose beads with bound pH-Countdown in Carmody buffers at several different pHs, with the same temperature and microscope settings.
</p>

<h3 id="mouse_imaging">
Mouse imaging</h3>
<p>
We generated <a href="https://www.atcc.org/products/crl-2638">CT26</a> cell lines expressing pH-Countdown using lentivirus infection with <a href="https://www.addgene.org/12260/">psPAX2</a>, <a href="https://www.addgene.org/12259/">pMD2.G</a> and either <a href="#sequences" target="_self">pCDH-pH-Countdown</a> or <a href="#sequences" target="_self">pCDH-mito-pH-Countdown</a>, and sorted via flow cytometry to select for the top 10% brightest cells.
</p>

<p>
We produced tumors under the skin of <a href="https://www.jax.org/strain/000651">BALB/cJ</a> mice by injecting 250000 CT26 cells in 50 &mu;L PBS under the flank skin. We imaged the mice on our <a href="#chicken_blaster" target="_self">custom fluorescence photography system</a> 1.5 weeks post-inoculation, before and after intravenous injection of 200 &mu;l of 0.75 mg/mL <a href="https://doi.org/10.1038/s41467-020-16298-2">BAM15</a> in 10% NMP/10% Kolliphor in the lateral tail vein. Mice were anaesthetized during imaging via isoflurane, temperature was maintained at 37&deg;C using a heating pad, and hair was removed using Nair before imaging. We calibrated the relationship between relaxation and pH by imaging pH-Countdown in Carmody buffers at several different pHs, with the same temperature and imaging settings.
</p>

<p>
All research was performed as part of Calico Life Sciences LLC AAALAC-accredited animal care and use program. All research and animal use in this study was approved by the Calico  Institutional Animal Care and use Committees (IACUC).
</p>

<h3 id="chicken_blaster">
Custom fluorescence photography setup</h3>
<img src="./data_generation/chicken_blaster/chicken_blaster.jpg" alt="A photograph of our simple fluorescence photography system" id="chicken_blaster_photo">
<p>
For large samples where microscopy was unneccesary, we acquired fluoresence photographs on a custom-built setup (detailed parts list below) controlled with a <a href="./data_generation/chicken_blaster/chicken_blaster.py">python script</a>. For imaging capillaries, we diluted 2 &mu;L of 35 mg/mL solution of purified pH-Countdown into 8 &mu;L of <a href="https://doi.org/10.1021/ed040pA386.1">Carmody buffers</a>, and loaded 5 &mu;L into capillary tubes later sealed with epoxy. A housing to prevent crosstalk was built using a silicone mat cut to fit the capillaries, and covered with black tape. Our set up is capable of delivering ~50 mW/cm<sup>2</sup> of blue light or violet light to a 10 cm Petri dish. We typically deliver ~500 &mu;s pulses of 10 mW/cm<sup>2</sup> cyan light during measurement, which yields substantial signal but minimal photoactivation.
</p>

<table>
  <tr>
    <th>Part</th>
    <th>Information</th>
  </tr>
  <tr>
    <td>Blue excitation filter</td>
    <td>Chroma et470/40</td>
  </tr>
  <tr>
    <td>Emission filter</td>
    <td>Chroma et525/50</td>
  <tr>
    <td>Violet excitation filter</td>
    <td>395/25</td>
  </tr>
  <tr>
    <td>Camera</td>
    <td>PCO edge 4.2 bi</td>    
  </tr>
  <tr>
    <td>Lens</td>
    <td>Zeiss Milvus 100mm F/2M ZF.2</td>    
  </tr>
  <tr>
    <td>Analog card</td>
    <td>PCIe-6738 Analog Output Device: 32 Analog Outputs (16-bit) (785822-01, NI)</td>    
  </tr>
  <tr>
    <td>Connector block</td>
    <td>SCB-68A Noise Rejecting, Shielded I/O Connector Block (782536-01, NI)</td>    
  </tr>
  <tr>
    <td>Daq cable</td>
    <td>SHC68-68-A2, 68-Pin Male VHDCI to 68-Pin Female SCSI, 1m Cable (157599-01, NI)</td>    
  </tr>
  <tr>
    <td>Output cables</td>
    <td>BNC Male to Unterminated Lead Cable 24 Inch Length Using RG58 Coax (PE33565-24, Pasternack)</td>    
  </tr>   
  <tr>
    <td>Rolling cart</td>
    <td>42X24X35 WELDED 3-SHELF CART (H-4156, Uline, Inc.)</td>    
  </tr>  
  <tr>
    <td>Breadboard</td>
    <td>Aluminum Breadboard, 600 mm x 900 mm x 12.7 mm, M6 Taps (MB6090/M, Thorlabs)</td>    
  </tr>  
  <tr>
    <td> Blue LED driver</td>
    <td>High-Power 1-Channel LED Driver with Pulse Modulation, 10.0 A Max, 50.0 V Max (DC2200, Thorlabs)</td>    
  </tr>  
  <tr>
    <td>Violet LED driver</td>
    <td>High-Power Driver for Solis® LEDs 10 A Max 14.0 V Max (DC20, Thorlabs)</td>    
  </tr>  
  <tr>
    <td>Blue LED</td>
    <td>High-Power LED for Microscopy, 470 nm (Blue), 3.0 W (Min) (SOLIS-470C, Thorlabs)</td>    
  </tr>   
  <tr>
    <td>Violet LED</td>
    <td>High-Power LED for Microscopy 405 nm (UV) 3.9 W (Min) (SOLIS-405C, Thorlabs) </td>    
  </tr> 
  <tr>
    <td>3 x Right-Angle Clamp</td>
    <td>Right-Angle Clamp for Ø25.0 mm Posts (RSA90/M, Thorlabs)</td>  
  </tr> 
  <tr>
    <td>Blue illumination lens</td>
    <td>f=100 mm &#8960;2" Achromatic Doublet SM2-Threaded Mount ARC: 400-700 nm (AC508-100-A-ML, Thorlabs) </td>    
  </tr> 
  <tr>
    <td>Base</td>
    <td>Rigid stand for 1.5 inch post (LPH200, Thorlabs)</td>    
  </tr> 
  <tr>
    <td>Base post</td>
    <td>1.5 inch post (P250/M, Thorlabs)</td>    
  </tr> 
  <tr>
    <td>&#8960;25 mm posts</td>
    <td>2 x RS300/M, RS150/M, 3 x RS100/M (Thorlabs)</td>    
  </tr> 
  <tr>
    <td>Pedestal Bases</td>
    <td>3 x BE1/M (Thorlabs)</td>    
  </tr>  
  <tr>
    <td>Cubes</td>
    <td>3 x RM1G (Thorlabs)</td>    
  </tr> 
  <tr>
    <td>Filter holders</td>
    <td>2 x SM1L05 (Throlabs)</td>    
  </tr>    
  <tr>
    <td>Extension tube</td>
    <td>SM2 Extension Tube (SM2E60, SM2L30, SM2L15, Thorlabs) </td>    
  </tr>  
  <tr>
    <td>SM2 adapter</td>
    <td>2 x SM2A6 (Thorlabs) </td>    
  </tr>   
  <tr>
    <td>Filter holders</td>
    <td>2 x SM1L05 (Thorlabs) </td>    
  </tr>                                   
</table>  

<h3 id="activation_due_to_measurement">
Disruptive vs non-disruptive imaging modes</h3>
<p>
pH-Countdown requires a relatively low dose of cyan light in order to activate. As a result, at high magnifications, where there are few fluorophores per pixel, it becomes challenging to take high SNR pictures that do not cause substantial activation. In this context, we image in a 'disruptive' mode, where we take 5-10 pictures that have good SNR and also cause activation, and image the decay at a single point after a waiting period on the order of 2 to 15 seconds, as illustrated by the blue dots and vertical blue lines in the lower left panel of <a href="./index.html#sensor_mechanism_comparison_figure">Figure 3</a>. This type of measurement is (slightly) instrument- and intensity-dependent, and absolute pH measurement typically requires a calibration. <a href="./index.html#autofluorescence_data_figure">Figure 4</a> shows an example of the 'disruptive' mode, using 50 ms exposures of 470&plusmn;12 nm, 1.7 W/cm<sup>2</sup> illumination.
</p>

<p>At low magnifications, where there are many fluorophores per pixel, it is straightforward to acquire a good SNR picture that does not cause significant activation. We call this the 'non-disruptive' imaging mode. Since imaging alone does not cause substantial photoactivation, we activate by interleaving high doses of cyan light in between images, and we typically record the whole decay trace after activation. This measures a relaxation rate that is a property of the fluorophore, and therefore instrument- and intensity-independent. <a href="./index.html#robust_against_background_figure">Figure 2</a> shows an example of the 'non-disruptive' mode, using 500 &mu;s exposures of 470&plusmn;20 nm, 10 mW/cm<sup>2</sup> illumination for imaging, and ~2 s exposures for activation.
 </p>

<h3 id="photosponging">
Photosponging</h3>
<p>We thank Adam Cohen for introducing us to the delightful concept of a photosponge: a light-driven binding/unbinding of a small molecule by an engineered protein or chemical, to manipulate intracellular concentration of that small molecule. We suspect that his idea may prove to be a very fun and useful tool in the future.
</p>

<h3 id="sequences">
Plasmids and sequences</h3>
<p>We intend to deposit these plasmids on AddGene. In the meantime, please contact Maria Ingaramo for plasmids.</p>

<p>Countdown AA sequence: MVSKGEENNMAVIKPDMKIKLRMEGSVNGHRFRIEGVGLGKPYEGKQSMDLKVKEGGPLPFAYDILTMAFCYGNRVFAK
YPENIVDYFKQSFPEGYSWERQMIYEDGGICVATNDITLDGDCMISEIRFKGVNFPANGPVFQKRTVKWELSHEKLYAR
DGLLYSDGNYALSLEGGGHYRCDNKTTYKAKKVVQLPDYHSVTHHIVIKSHDKDYSNVNLHEHAEAHSELPRQAMDELYK
</p>

<p>pH-Countdown AA sequence: MVSKGEENNMAVIKPDMKIKLRMEGSVNGHRFRIEGVGLGKPLEGKQSMDLKVKEGGPLPFAYDILTMAFCYGNRVFAK
YPENIVDYFKQSFPEGYSWERQMIYEDGGICVATNDITLDGDCMISEIRFKGVNFPANGPVFQKRTVKWELSHEKLYAR
DGLLYSDGNYALSLEGGGHYRCDNKTTYKAKKVVQLPDYHWVTHSIVIKSHDKDYSNVNLHEHAEAHSELPRQAMDELYK
</p>
<p>
<a href="./sequences/pQE31_countdown.dna">pQE31-6xHis-Countdown plasmid Snapgene file</a> (<em>E. coli</em> plasmid encoding Countdown with a hexahistidine tag; <a href="./sequences/pQE31_countdown.txt">plain text format</a>)
</p>
<p>
<a href="./sequences/pQE31_pHcountdown.dna">pQE31-6xHis-pHCountdown plasmid Snapgene file</a> (<em>E. coli</em> plasmid encoding pH-Countdown with a hexahistidine tag; <a href="./sequences/pQE31_pHcountdown.txt">plain text format;</a> <a href="./sequences/pHcountdown_coding.png">coding sequence snapshot</a>)
</p>

<p>
<a href="./sequences/pCDH EF1 pH-countdown.dna">pCDH-EF1-pHCountdown plasmid Snapgene file</a> (lentiviral plasmid encoding pH-Countdown; <a href="./sequences/pCDH EF1 pH-countdown.txt">plain text format</a>)
</p>
<p>
<a href="./sequences/pCDH EF1 Mito-pH-countdown.dna">pCDH-EF1-Mito-pHCountdown plasmid Snapgene file</a> (lentiviral plasmid encoding pH-Countdown targeted to mitochondria via fusion to the COX8A presequence; <a href="./sequences/pCDH EF1 Mito-pH-countdown.txt">plain text format</a>)
</p>

<p>
<a href="./sequences/N1_Mito_pHcountdown.dna">N1-Mito-pHCountdown plasmid Snapgene file</a> (mammalian cell plasmid encoding pH-Countdown targeted to mitochondria via fusion to the COX8A presequence; <a href="./sequences/N1_Mito_pHcountdown.txt">plain text format</a>)
</p>
<p>
<a href="./sequences/eft_pHcountdown_unc54.dna">pUC57-Peft-pHCountdown-unc54 plasmid Snapgene file</a> (<em>C. elegans</em> plasmid encoding pH-Countdown with three introns; <a href="./sequences/eft_pHcountdown_unc54.txt">plain text format</a>)
</p>

<p>
<a href="./sequences/pRSET_calcium_sensor.dna">pRSET-TorA-6xHis-M13-pHCountdown-Calmodulin plasmid Snapgene file</a> (<em>E. coli</em> plasmid encoding a fusion between pH-Countdown and the calcium-binding domains of <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3560286/">GECO 1.2</a> with a hexahistidine tag, targetted to the periplasm via the TorA presequence. This fusion has a calcium-dependent relaxation rate; <a href="./sequences/pRSET_calcium_sensor.txt">plain text format</a>)
</p>

<noscript>
<h3>Your browser doesn't seem to support Javascript. This document uses Javascript to generate a reference list, which would normally be displayed here. Either activate Javascript, or use the "Download PDF" link above if you want to see the reference list.</h3></noscript>

</section>
<footer>
  <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="./images/cc_by_4p0.png"></a>

  <p><small>Hosted on <a href="https://pages.github.com">GitHub Pages</a></small></p>
</footer>
</div>
 <!--[if !IE]><script>fixScale(document);</script><![endif]-->
</body>
</html>
 
